### PR TITLE
Implement lintian support

### PR DESCRIPTION
**Pull request by waja**
This PR adds [lintian](https://lintian.debian.org/) support, which helps to build packages with higher quality standards. I would really love to have that included.

### CODE REVIEW

**Comment by dawidd6** [comment]
This is great, thanks. Just thinking maybe we should hide this functionality behind some new input? Let's say `run_lintian: true/false`, defaulting to `false`.

File: main.js [hunk 1]
```diff
@@ -38,6 +38,8 @@ async function main() {
         const sourceRelativeDirectory = core.getInput("source_directory") || "./"
         const artifactsRelativeDirectory = core.getInput("artifacts_directory") || "./"
         const osDistribution = core.getInput("os_distribution") || ""
+        const lintianRun = core.getInput("lintian_run") === 'false'
```
Comment (dawidd6) [inline main.js:41]: Try `getBooleanInput`, should be available.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -51,6 +60,7 @@ jobs:
           cpu_architecture: ${{matrix.arch}}
           source_directory: ${{matrix.package}}
           artifacts_directory: artifacts
+          lintian_opts: ${{matrix.lintian_opts}}
```
Comment (dawidd6) [inline .github/workflows/test.yml:63]:
```suggestion
          lintian_opts: ${{matrix.lintian_opts}}
          lintian_run: ${{matrix.lintian_run}}
```

File: main.js [hunk 3]
```diff
@@ -38,6 +38,8 @@ async function main() {
         const sourceRelativeDirectory = core.getInput("source_directory") || "./"
         const artifactsRelativeDirectory = core.getInput("artifacts_directory") || "./"
         const osDistribution = core.getInput("os_distribution") || ""
+        const lintianOpts = core.getInput("lintian_opts") || ""
+        const lintianRun = core.getBooleanInput('lintian_run') || 'false'
```
Comment (dawidd6) [inline main.js:42]:
```suggestion
        const lintianRun = core.getBooleanInput('lintian_run') || false
```

**Review by dawidd6 — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -51,6 +60,8 @@ jobs:
...
+          lintian_run: ${{matrix.lintian_run}}
```
Comment (dawidd6) [inline .github/workflows/test.yml:64]:
```suggestion
          lintian_run: ${{matrix.lintian_run || 'false' }}
```

Maybe this will work

**Comment by waja** [comment]
okay ... the CI succeeded for micro and dropbear ... all the other once are failing cause other reasons not related to this change

thanks for helping me solving this

**Comment by dawidd6** [comment]
Thanks!

**Pull request merged**